modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -128,6 +128,8 @@ private var credentialStore = SynchronizedDictionary<Int, URLCredential>()
                 encoding = FormDataEncoding(contentTypeForFormPart: contentTypeForFormPart(fileURL:))
             } else if contentType.hasPrefix("application/x-www-form-urlencoded") {
                 encoding = FormURLEncoding()
+            } else if contentType.hasPrefix("application/octet-stream"){
```

Comment:
@aniruddhJo Hi, thanks for creating this PR.
Could you please also add support for `application/octet-stream` in alamofire please?
Thanks

Reply:
created a ticket to track this: https://github.com/OpenAPITools/openapi-generator/issues/16300